server.js

```diff
@@ -70,14 +70,17 @@ if (config.SENTRY.DSN) {
     dsn: config.SENTRY.DSN,
     environment: config.SENTRY.ENVIRONMENT,
     release: config.SENTRY.RELEASE,
-    integrations: [
-      // enable HTTP calls tracing
-      new Sentry.Integrations.Http({ tracing: true }),
-      // enable Express.js middleware tracing
-      new Tracing.Integrations.Express({
-        app,
-      }),
-    ],
+    integrations:
+      config.SENTRY.ENVIRONMENT === 'production'
+        ? [
+            // enable HTTP calls tracing
+            new Sentry.Integrations.Http({ tracing: true }),
+            // enable Express.js middleware tracing
+            new Tracing.Integrations.Express({
+              app,
+            }),
+          ]
+        : [],
     // Half of all requests will be used for performance sampling
     tracesSampleRate: 0.5,
```

Comment:
I don't know whether we might need to do this one as well? Could use a similar conditional and if it's not production set to 0 or something.

Reply:
Although I have a feeling this isn't relevant without the other integrations.